packages/flutter_tools/test/general.shard/macos/xcode_validator_test.dart

```diff
@@ -58,6 +59,19 @@ void main() {
       expect(result.messages.last.message, contains('Flutter requires a minimum Xcode version of 12.0.1'));
     });
 
+    testWithoutContext('Check for Xcode version in non-verbose flutter doctor', () async {
```

Comment:
> Also, the xcode validator doesn't have a concept of doctor verbosity, you're just checking that the result status info is correct.

See my comment in https://github.com/flutter/flutter/pull/87022#discussion_r677692728, this test is the same case as `Succeeds when all checks pass` so it doesn't add any test coverage.  Do you can remove it, you're covered by the new `statusInfo` expectations.

If you want you can add the `ValidationMessage` test expectations I already wrote in that comment, but that's optional.